### PR TITLE
Chef 17 compatibility

### LINKED ISSUE #10: chef_ca Chef 17 Compatability

### PR DESCRIPTION
**Pull request by MarkGibbons**
# Description
Chef 17 support

## Issues Resolved

Resolves #10 

## Check List

- [ ] All tests pass. See TESTING.md for details.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable.

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
-Change log for chef_ca
+# Change log for chef_ca
 
-0.1.0
+## unreleased
```
Comment (ramereth) [inline CHANGELOG.md:3]:
```suggestion
## Unreleased
```

Comment (MarkGibbons) [reply CHANGELOG.md:3]: fixed.

**Review by damacus — APPROVED** [review]

**Comment by damacus** [comment]
FYI, I've moved to main branch on this repo now

**Comment by kitchen-porter** [comment]
Released as: [1.0.0](https://github.com/sous-chefs/chef_ca/releases/tag/1.0.0)

**Pull request merged**